core/domain/suggestion_registry.py

```diff
@@ -1223,6 +1224,51 @@ def set_translation_suggestion_count_for_language_code(
         """
         self.translation_suggestion_counts_by_lang_code[language_code] = count
 
+    def are_translation_reviewers_needed_in_lang_code(self, lang_code):
```

Comment:
This reads weirdly. Perhaps are_translation_reviewers_needed_for_lang_code(...)

Reply:
Changed